lib/validators/passwordValidator.ts

```diff
@@ -0,0 +1,9 @@
+import * as Yup from 'yup'
+
+export const passwordValidator = Yup.string()
+  .required('Passwort muss angegeben werden.')
+  .max(128, 'Das Passwort darf nicht länger als 128 Zeichen sein.')
+  .matches(
+    /^(?=.*[A-Za-z])(?=.*\d)(?=.*[$&+,:;=?@#|'<>.^*()%!-[\]{}])[A-Za-z\d$&+,:;=?@#|'<>.^*()%!-[\]{}]{8,128}$/,
```

Comment:
Please tell me you did not come up with this regex yourself :smile:

Reply:
Oh I see you only moved it, and did not introduce it. I wonder where it comes from though

Reply:
I bet it's Stackoverflow 🙈